### DISCUSSION TITLE
Fleet carrier support?

### DISCUSSION THREAD

**Discussion by marcaddeo** [iaincollins/icarus#7] [Ideas & Suggestions]
Would love to see fleet carriers be included in the navigation, but I know that can make the nav list get real long.

But it's nice to be able to swing over to a FC that's 1Ls away to refuel/repair/restock.

**Comment by ayaanrauf** [comment]
Maybe it could be toggleable?

**Reply by iaincollins** [reply]
Agree with both of these comments!

I currently get the data for them but then hide Fleet Carriers as there are so many, but am very happy to make this a toggle. I could maybe add some icons to indicate what services are available.

EDSM makes it possible to also get them for remote systems, but if I did that I should probably include the time since they were last recorded (e.g. 1 hour ago / 3 days ago / 2 weeks ago) and display that in the UI, given they move around.

I am thinking of also having a toggle that goes between the current view, to an expanded view where each row is taller (maybe 2-3x taller) and has detailed but concise information about planet surface, station / ship services. My primary motivation here was to improve things for explorers but it might be useful to display expanded info for stations / ships / settlements too.
